migrations/20241030234724_first_migration.sql

```diff
@@ -0,0 +1,21 @@
+-- +goose Up
+-- create "metrics" table
+CREATE TABLE "metrics" (
+  "id" character varying(255) NOT NULL,
```

Comment:
[CHANGE REQUEST] It should be `int` of `uuid`.

Reply:
MetricName is the primary key and is it a string or do you want me to come up with a new identifier field?